app/views/ScreenLockConfigView.js

```diff
@@ -0,0 +1,281 @@
+		try {
+			this.serverRecord = await serversCollection.find(server);
+			this.setState({
+				autoLock: this.serverRecord?.autoLock,
+				autoLockTime: this.serverRecord?.autoLockTime || 1800,
+				biometry: this.serverRecord.biometry === null ? true : this.serverRecord.biometry
+			});
+		} catch (error) {
+			// Do nothing
+		}
+
+		const biometryLabel = await supportedBiometryLabel();
+		this.setState({ biometryLabel });
+	}
+
+	save = async() => {
+		const { autoLock, autoLockTime, biometry } = this.state;
+		const serversDB = database.servers;
+		await serversDB.action(async() => {
+			await this.serverRecord?.update((record) => {
+				record.autoLock = autoLock;
+				record.autoLockTime = autoLockTime;
+				record.biometry = biometry === null ? true : biometry;
+			});
+		});
+	}
+
+	setPasscode = async() => {
+		const { autoLock } = this.state;
+		const { navigation } = this.props;
+		if (autoLock) {
+			const storedPasscode = await RNUserDefaults.get(PASSCODE_KEY);
+			if (!storedPasscode) {
+				navigation.navigate('ChangePasscodeView', { forceSetPasscode: true });
+			}
+		}
+	}
+
+	autoLock = () => {
+		this.setState(({ autoLock }) => ({ autoLock: !autoLock }), () => {
+			this.save();
+			this.setPasscode();
+		});
+	}
+
+	toggleBiometry = () => {
+		this.setState(({ biometry }) => ({ biometry: !biometry }), () => this.save());
+	}
+
+	isSelected = (value) => {
+		const { autoLockTime } = this.state;
+		return autoLockTime === value;
+	}
+
+	changeAutoLockTime = (autoLockTime) => {
+		this.setState({ autoLockTime }, () => this.save());
+	}
+
+	changePasscode = () => {
+		const { navigation } = this.props;
+		navigation.navigate('ChangePasscodeView');
+	}
+
+	renderSeparator = () => {
+		const { theme } = this.props;
+		return <Separator theme={theme} />;
+	}
+
+	renderIcon = () => {
+		const { theme } = this.props;
+		return <CustomIcon name='check' size={20} color={themes[theme].tintColor} />;
+	}
+
+	renderItem = ({ item }) => {
+		const { theme } = this.props;
+		const { title, value } = item;
+		return (
+			<>
+				<ListItem
+					title={title}
+					onPress={() => this.changeAutoLockTime(value)}
+					right={this.isSelected(value) ? this.renderIcon : null}
+					theme={theme}
+				/>
+				<Separator theme={theme} />
+			</>
+		);
+	}
+
+	renderAutoLockSwitch = () => {
+		const { autoLock } = this.state;
+		return (
+			<Switch
+				value={autoLock}
+				trackColor={SWITCH_TRACK_COLOR}
+				onValueChange={this.autoLock}
+			/>
+		);
+	}
+
+	renderBiometrySwitch = () => {
+		const { biometry } = this.state;
+		return (
+			<Switch
+				value={biometry}
+				trackColor={SWITCH_TRACK_COLOR}
+				onValueChange={this.toggleBiometry}
+			/>
+		);
+	}
+
+	renderAutoLockItems = () => {
+		const { autoLock } = this.state;
+		const { theme } = this.props;
+		if (!autoLock) {
+			return null;
+		}
+		return (
+			<>
+				<View style={{ height: 36 }} />
+				<Separator theme={theme} />
```

Comment:
`<Separator style={{ marginTop: 36 }} theme={theme} />`, then you can remove the void view above.